.github/workflows/pr_update.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PR update
+
+on:
+  push:
+    branches:
+      - "main"
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Automatically update PR
+        uses: adRise/update-pr-branch@v0.6.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          base: "main"
+          required_approval_count: 1
+          require_passed_checks: true
+          sort: "created"
+          direction: "desc"
```

Comment:
I'm ok trying this either way, _but_: I do wonder if `asc` would make more sense here.  Then we would consistently grab the oldest-created PR in the queue. My logic here is that big or more complex PRs have often sat for longer anyways, and also have a higher chance of being invalidated by other commits (think large refactors, or that gRPC PR we had a couple of months ago).

The counter-argument is that grabbing the newest could help for "I need to unwedge CI quickly" style PRs. But I think we can hand manage those if necessary.

Reply:
changed the direction to `asc`.